app/chapters/[slug]/genesis/info.tsx

```diff
@@ -0,0 +1,31 @@
+import { BoxButton } from 'components/ui/BoxButton'
+
+export const Info = ({
+  isSmallScreen,
+  genesis
+}: {
+  isSmallScreen: boolean,
```

Comment:
This doesn't seem used.

Reply:
I removed that line.

I also think it's a bit overkill to add a dedicated library for this. Just trying to do things the react way, and this is what was recommended in the blog posts I found.